news/_posts/2020-05-13-the-new-order-pages-in-prestaShop-1.7.7.md

```diff
@@ -0,0 +1,215 @@
+---
+layout: post
+title: "The new order pages in PrestaShop 1.7.7"
+subtitle: "Introducing the revamp of order pages"
+date: 2020-05-12 12:00:00
+authors: [MateusShirlaw]
+icon: icon-leaf
+tags:
+- 1.7.7
+- version
+- minor
+- product
+- ux
+---
+
+With the release of the new version 1.7.7, this article has the ambition to detail one of the major features of the version. Our work was focused on one of the most used pages by our merchants: the order page.
+
+We are thrilled to introduce to you the new feature around the order process. But before jumping on the features that await the merchant in the new version, we&#39;d like to share our thinking process.
+
+### ***First, why rethinking this page in particular?***
```

Comment:
Before using h3 you need a h2. Search engine don't like it otherwise

Reply:
if you don't think it has a h2 importance, then I recommend putting the titles in bold only